beehive.go

```diff
@@ -90,12 +89,23 @@ func main() {
 
 	config := cfg.Config{}
 	var err error
-	if cfg.Exist(configFile) {
-		config, err = cfg.LoadConfig(configFile)
-		if err != nil {
-			log.Panicf("Error loading config file %s!: %v", configFile, err)
+	if configFile == "" {
```

Comment:
Related to the previous comment (I totally understand why you removed the default): maybe we can check whether the flag has been "touched" (as in passed in by the command)?

If it's untouched we can proceed trying to find the config file here.

Maybe I'm overthinking this though, and your change is just fine.

Reply:
Done :heavy_check_mark: